src/plugins/kibana_react/public/code_editor/code_editor.tsx

```diff
@@ -150,14 +145,15 @@ export class CodeEditor extends React.Component<Props, {}> {
     monaco.editor.defineTheme('euiColors', this.props.useDarkTheme ? DARK_THEME : LIGHT_THEME);
```

Comment:
Should this be set on the default package?

Reply:
Not a blocker to this PR since this is the current implementation.

Reply:
We should avoid putting any code into the `@kbn/ui-shared-deps` that isn't directly from npm as that module is only built on bootstrap. So I'm going to say we should leave it here.